schema.graphql

```diff
@@ -108,17 +134,68 @@ type MiscCostNode implements Node {
   value: Float
 }
 
+type Mutation {
+  register(email: String!, password1: String!, password2: String!): Register
```

Comment:
Want to also collect first and last name here?

Reply:
Also... If we are verifying emails, we shouldn't be able to get a token from the register mutation